### ISSUE TITLE
Struck

### ISSUE THREAD

**Issue by Nayeem4** [spyder-ide/spyder#6459]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "C:\Users\Binary\Anaconda3\lib\site-packages\spyder\plugins\ipythonconsole.py", line 906, in <lambda>
Traceback (most recent call last):
  File "C:\Users\Binary\Anaconda3\lib\site-packages\spyder\plugins\ipythonconsole.py", line 906, in <lambda>
    clear_variables))
  File "C:\Users\Binary\Anaconda3\lib\site-packages\spyder\plugins\ipythonconsole.py", line 948, in execute_code
    sw.sig_prompt_ready.disconnect()
TypeError: disconnect() failed between 'sig_prompt_ready' and all its connections
    clear_variables))
  File "C:\Users\Binary\Anaconda3\lib\site-packages\spyder\plugins\ipythonconsole.py", line 948, in execute_code
    sw.sig_prompt_ready.disconnect()
TypeError: disconnect() failed between 'sig_prompt_ready' and all its connections

## Version and main components

* Spyder Version: 3.2.4 
* Python Version: 3.6.3
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.3 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.26.1 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.1.0 (OK)
pylint >=0.25    :  1.7.4 (OK)

```

**Comment by ccordoba12** [comment]
This is fixed in our latest version (3.2.6). Please update.

**Comment by CAM-Gerlach** [comment]
For the record, duplicate of #5748 . Also, you can update with ``conda update spyder`` .
